app/views/provider/admin/dashboards/_services_filter.html.slim

```diff
@@ -0,0 +1,20 @@
+#api_filter
+
+javascript:
+  document.addEventListener('DOMContentLoaded', function () {
+    var allApis = #{json @services}
+
+    var displayApis = function(filteredApis) {
+      for (api = 0; api < allApis.length; api++) {
+        document.getElementById("service_" + allApis[api].service.id).style.display = "none"
+      }
+      for (api = 0; api < filteredApis.length; api++) {
```

Comment:
Can we avoid doing second loop?
including an if in line 9?

Reply:
Lets leave this for next iteration @josemigallas

Reply:
Problem with that is performance @hallelujah -> http://jsben.ch/XPYGU it's simply much quicker to iterate once each array than looking inside `filteredApis` each time.